### PR TITLE
Week 3

### PR DESCRIPTION
**Pull request by nuhaq**

### CODE REVIEW

File: client/src/components/BookDetail/BookDetail.jsx [hunk 1]
```diff
@@ -138,7 +138,7 @@ export default function BookDetail(props) {
                     {/* <input className="rating-type" type="radio" value="1"/> */}
                     <input type="submit" value="Send" onClick={(e) => {e.preventDefault(); addReview()}}/>
                 </form>
-                <div className="ratings">{reviews.reverse().map(r => {
+                <div className="ratings">{reviews.slice().reverse().map(r => {
```
Comment (helpark) [inline client/src/components/BookDetail/BookDetail.jsx:128]: Hey Nuha, what exactly are you testing here?

File: backend/app.js [hunk 2]
```diff
@@ -37,16 +37,13 @@ app.post('/logout', async (req, res) => {
         {useMasterKey: true}
       )
       .then ( function(res) {
-        console.log("success")
```
Comment (helpark) [inline backend/app.js:40]: thanks for removing these!

**Review by helpark — APPROVED** [review]
General comment, THanks for working on this PR. So far looks great. Please use descriptive and specific commit messages and do only what you described. 

For example i see you did some clean up work here, and some local storage setup. This could have been 2 separate PRs, 1. for cleanup, 2. for setting up local storage.

File: client/src/components/BookDetail/BookDetail.jsx (hunk 1 above)
Comment (nuhaq) [reply client/src/components/BookDetail/BookDetail.jsx:128]: hi! i was just doing a test commit to make sure I could create the PR properly--I'll use more descriptive commit names from now on.

**Pull request merged**